src/tokens.scss

```diff
@@ -0,0 +1,7 @@
+:host {
+  --ds-auro-input-background-color: var(--ds-color-background-primary-100-default, $ds-color-background-primary-100-default);
+  --ds-auro-input-border-color: var(--ds-color-border-primary-default, $ds-color-border-primary-default);
+  --ds-auro-input-text-color: var(--ds-color-text-secondary-default, $ds-color-text-secondary-default);
+  --ds-auro-input-alert-icon-color: var(--ds-color-alert-error-default, $ds-color-alert-error-default);
+  --ds-auro-input-type-icon-color: var(--ds-color-text-disabled-default, $ds-color-text-disabled-default);
```

Comment:
I see input text color (I assume thats for the value in the actual HTML5 input) but we also need to be able to style the label text color and the help text color.

It looks like you are using the same token value for all of them but it should be 3 different ones.